build.yaml

```diff
@@ -10,12 +10,24 @@ cassandra:
 build:
   - type: maven
     version: 3.2.5
-    goals: clean compile test --fail-never -Plong
+    goals: clean compile test integration-test --fail-never -Plong
     properties: |
       com.datastax.driver.TEST_BASE_NODE_WAIT=120
       com.datastax.driver.NEW_NODE_DELAY_SECONDS=100
       cassandra.version=$CCM_CASSANDRA_VERSION
       ccm.java.home=$CCM_JAVA_HOME
+      failIfNoTests=false
```

Comment:
The only reasons I have 'failIfNoTests' and 'maven.test.failure.ignore' present here is so the build can continue and run other tests.  For example, if the 'driver-core' build fails tests, the artifacts aren't created and thus integration tests fail.  We'd normally not want to use these flags for build/releases but for the intent of CI testing it works.